### ISSUE TITLE
Request for 128 Mel Band Support and Training Insights

### ISSUE THREAD

**Issue by TobiasPfeifer** [Plachtaa/seed-vc#192]
Hi! First off, excellent work on Seed-VC! The quality of your approach is really impressive.

**Main Question**
I'm wondering if there's a pretrained model available (or planned) that supports 128 mel bands as input for 44kHz BigVGAN?

**Background & Technical Issue**
I attempted to adapt the existing pretrained models to work with 128 mel bands, but ran into the expected shape mismatch errors:

```
estimator.x_embedder.parametrizations.weight.original1: copying a param with shape torch.Size([512, 80]) from checkpoint, the shape in current model is torch.Size([512, 128])
```
This makes sense since the current models are hardcoded for 80 mel band inputs.

If a 128-band model isn't available, would you be willing to share insights on the compute costs for the 80 mel band pretrain, or provide estimates for scaling to 128 bands? Also, have you evaluated different hardware configurations for training? During fine-tuning with 2xA40 GPUs, I observed that higher batch sizes actually increase training time per epoch (even when well below memory limits), suggesting that fewer iterations with larger batches don't scale linearly despite sufficient VRAM availability.

I'd be willing to contribute both technically and financially to develop a 128 mel band pretrained snapshot that surpasses current performance on both speech and singing voice conversion tasks.

I'm exploring adapter-based architecture expansion to accept 128 mel band inputs while preserving existing 80-band learned representations. This would use LoRA modules to extend input embedding layers from 80→128 dimensions, potentially with auxiliary pathways that process the full 128-band resolution whilemaintaining compatibility with frozen pretrained weights.

Would this progressive expansion approach be technically feasible with Seed-VC's architecture? I'm eager to discuss implementation details and potential collaboration.

Thanks for your time and for making this fantastic work available to the community!

**Comment by Plachtaa** [comment]
Hi, Thanks for your interest. 
I am not sure whether you can use LoRA modules to adapt a 80 mel band model to 128 mel band while swapping out embeddings and projection layers. You can launch a experiment to see it works, but I guess it's not gonna be easy because model must learn projection from scratch
